components/api/src/error.rs

```diff
@@ -14,10 +14,99 @@
 
 //! The set of errors for the Exonum API module.
 
+use actix_web::http::StatusCode;
```

Comment:
Nit: Since this type is required to parse `ApiError`, it's better to re-export this type via `pub use`, so clients won't have to import `actix_web` just to get this type available.